skimage/measure/block.py

```diff
@@ -2,8 +2,10 @@
 from ..util import view_as_blocks
 
 
-def block_reduce(image, block_size, func=np.sum, cval=0):
-    """Down-sample image by applying function to local blocks.
+def block_reduce(image, block_size, func=np.sum, cval=0, **func_kwargs):
+    """
+    Down-sample image by applying function to local blocks. Able to do
+    max and mean pooling.
```

Comment:
This addition should be in a new line. Please see [PEP257](https://www.python.org/dev/peps/pep-0257/) for a precise guide. ie:

```python
"""Downsample image by applying function `func` to local blocks.

This function is useful for max and mean pooling, for example.

Parameters
----------
...
"""
```